src/config.rs

```diff
@@ -25,22 +24,16 @@ lazy_static! {
     ///
     /// Examples:
     /// ```
-    ///
     /// let config_arc = Arc::clone(&CONFIG);
     /// let mut config_lock = config_arc.lock();
-    /// while config_lock.is_err() {
-    ///     dbg!("Could not acquire config lock. Waiting...");
-    ///     sleep(Duration::from_millis(5));
-    ///     config_lock = config_arc.lock();
-    /// }
     /// let value = match config_lock {
     ///     Ok(config) => {
     ///         // ACCESS CONFIG FIELDS HERE AND COPY THEM INTO VALUE
     ///         // Example: String::from(&config.autokommentator.token)
     ///     },
-    ///     Err(_) => {
-    ///         println!("Something went wrong internally...");
-    ///         return
+    ///     Err(why) => {
+    ///         eprintln!("Something went wrong internally: {:?}\nMutex is poisoned: {}", why, why);
+    ///         return;
```

Comment:
This should panic instead of just returning.
A Mutex is poisened when the thread that has it locked panics. It becomes unusable afterwards.
```suggestion
    ///         panic!("Something went wrong internally: {:?}\nMutex is poisoned: {}", why, why);
```

Reply:
0911b0d405d1fedbb5d8e5221c53f3fa56a5ebc7